internal/mvdan.cc/gogrep/parse.go

```diff
@@ -308,8 +308,7 @@ func (m *matcher) tokenize(src []byte) ([]fullToken, error) {
 		}
 		toks = append(toks, wt)
 		if caseStat == caseHere {
-			toks = append(toks, fullToken{wt.pos, token.COLON, ""})
-			toks = append(toks, fullToken{wt.pos, token.IDENT, "gogrep_body"})
+			toks = append(toks, fullToken{wt.pos, token.COLON, ""}, fullToken{wt.pos, token.IDENT, "gogrep_body"})
```

Comment:
I think this is the only change that should be done. All other cases are for unit test purpose.

Reply:
I prepared a complete gogrep rewrite in my local branch, so we could merge this line, but it would be removed in a few days. :)